app/vep/models/client_model.py

```diff
@@ -1 +1,38 @@
+from pydantic import BaseModel, Field
+from typing import List, Optional
 
+class FieldOptions:
+    GENCODE_LABEL: str = "GENCODE"
+    GENCODE_VALUE: str = "gencode_comprehensive"
+
+class Options(BaseModel):
+  label: str = FieldOptions.GENCODE_LABEL
+  value: str = FieldOptions.GENCODE_VALUE
+
+class TranscriptSet(BaseModel):
+    label: str
+    description: str = None
+    type: str = "select"
+    options: List[Options] = [Options()]
+    default_value: str = FieldOptions.GENCODE_VALUE
```

Comment:
Transcript set is genome-specific; so it doesn't make sense to have a default value

Reply:
Isn't `default_value` that tells you the default from a list of options?

Reply:
Oh, sorry, my bad, you are right. The transcript set, for the foreseeable future, will only contain a single option, and the default value should be the value of that option.

Since the client won't send transcript set's value back to the server (at least not unless we introduce the ability to choose between different transcript sets), it does not _really_ matter what the actual value is. It could be an empty string. Or it could be a slightly more meaningful string such as "default_transcript_set" or just "default". Or it could be generated dynamically from the provider name, version, and date, which is probably an overkill for now.

Reply:
I am generating the label with provider name and version. So I can use the same for `value` and `default_value`